scala3doc/src/dotty/dokka/translators/ScalaContentBuilder.scala

```diff
@@ -248,6 +248,14 @@ class ScalaPageContentBuilder(
 
     def signature(d: Documentable) = addChildren(signatureProvider.signature(d).asScala.toList)
 
+    private def buildSignature(d: Documentable, s: Signature) = signatureProvider.asInstanceOf[ScalaSignatureProvider].signature(d, s)
```

Comment:
I think we can move `signature` to `ScalaSignatureProvider` object.

Reply:
We are using contentBuilder field from ScalaSignatureProvider class which will be hard to obtain in object